hummingbot/connector/connector_status.py

```diff
@@ -0,0 +1,32 @@
+#!/usr/bin/env python
+
+connector_status = {
+    'binance': 'green',
+    'binance_us': 'yellow',
+    'bitfinex': 'green',
+    'bittrex': 'yellow',
+    'celo': 'green',
+    'coinbase_pro': 'green',
+    'crypto_com': 'yellow',
+    'eterbase': 'red',
+    'ethereum': 'red',
+    'huobi': 'green',
+    'kraken': 'yellow',
+    'kucoin': 'green',
+    'liquid': 'green',
+    'loopring': 'green',
+    'okex': 'red'
+}
+
```

Comment:
You can create another list here similar to `connector_status` but for warning messages, and create another function to retrieve it similar to `get_connector_status`, the only different is it returns None if the connector is not on the list to indicate no warning.

Reply:
Thanks @Nullably! I just pushed a new commit with these changes.